packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -753,3 +754,55 @@ export function isPostPriorityEnabled(state: GlobalState) {
         getConfig(state).PostPriority === 'true'
     );
 }
+
+export const canWrangler = createSelector(
```

Comment:
Remember that this should have a `makeCanWrangle` factory since it takes arguments